website/client/src/components/userMenu/profile.vue

```diff
@@ -5,16 +5,18 @@
   >
     <div class="header">
       <div class="profile-actions">
-        <button
-          v-b-tooltip.hover.left="$t('sendMessage')"
-          class="btn btn-secondary message-icon"
-          @click="sendMessage()"
-        >
-          <div
-            class="svg-icon message-icon"
-            v-html="icons.message"
-          ></div>
-        </button>
+        <router-link :to="{ path: '/private-messages', query: { uuid: user._id } }">
+          <button
+            v-b-tooltip.hover.left="$t('sendMessage')"
+            class="btn btn-secondary message-icon"
+            @click="sendMessage()"
```

Comment:
I think having a simple click executes the button click and the router-link which is then the reason the modals are opening up again.

Reply:
`sendMessage` should be probably removed

Reply:
So I think the button and the `sendMessage ` method can be removed